test/celeritas/optical/RayleighMfpCalculator.test.cc

```diff
@@ -68,40 +34,44 @@ class RayleighMfpCalculatorTest : public MockImportedData
 // Check calculated MFPs match expected ones
 TEST_F(RayleighMfpCalculatorTest, mfp_table)
 {
-    static std::vector<std::vector<real_type>> expected_tables
-        = {{1189584.7068151, 682569.13017288, 343507.60086802},
-           {12005096.767467, 6888377.4406869, 3466623.2384762},
-           {1189584.7068151, 277.60444893823},
-           {11510.805603078, 322.70360179716, 4.230373664558},
-           {12005096.767467, 2801.539271218}};
-
-    auto core_materials = this->build_core_materials();
-
-    for (auto opt_mat : range(OpticalMaterialId(import_materials().size())))
+    static constexpr auto expected_mfps
+        = expressed_as<units::CmLength>(1189584.7068151,
+                                        682569.13017288,
+                                        343507.60086802,
+                                        12005096.767467,
+                                        6888377.4406869,
+                                        3466623.2384762,
+                                        1189584.7068151,
+                                        277.60444893823,
+                                        11510.805603078,
+                                        322.70360179716,
+                                        4.230373664558,
+                                        12005096.767467,
+                                        2801.539271218);
```

Comment:
Adding a trailing comma will make clang-format indent this more reliably...
```suggestion
                                        2801.539271218,);
```

Reply:
Because it's a function argument it doesn't support trailing commas. I added an option for mapping `Array`s so that a brace-initializer can be passed as an argument, which has better clang-format indenting.